php/Slim4OtelDropIn/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Extend the OpenTelemetry PHP base image
+FROM ghcr.io/open-telemetry/opentelemetry-php/opentelemetry-php-base:8.1
+
+USER root
+
+# Set the DD_TRACE_VERSION to the latest stable version
+ENV DD_TRACE_VERSION=1.3.0
```

Comment:
Do we have to use an env var there? Can't we use `latest` for instance?

Reply:
No we don't, I also prefer latest so updated here: https://github.com/DataDog/trace-examples/pull/126/commits/e501215b5c193dfce3e7c94d9f4f6680fa38449b